src/main/java/uk/gov/pay/api/service/RefundsParams.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.pay.api.service;
+
+public class RefundsParams {
+
+    private String page;
+    private String displaySize;
+
+    public RefundsParams(String page, String displaySize) {
+        this.page = page;
+        this.displaySize = displaySize;
+    }
+
+    public void setPage(String page) {
```

Comment:
I would personally not put setters in unless you plan to use them. It's good to have "immutable" classes, especially considering that this should be instantiated after the parameters are parsed and never changed

Reply:
✪🌟⭒💫✶✨✴︎

Reply:
was this the response you were talking about? it's just a list of emojis 🤔